static/content.json

```diff
@@ -58,11 +63,11 @@
                     "websiteUrl": "https://bankid.no"
                 },
                 {
-                    "logoPath": "/images/fdir.svg",
-                    "backgroundPath": "/images/client-background-2.svg",
-                    "name": "Fiskeridirektoratet",
-                    "text": "The Directorate of Fisheries is a branch of the Norwegian government that is responsible for managing Norway's marine resources. Savvy has been a part of a team that's been developing a new system for managing fishing quotas.",
-                    "websiteUrl": "https://fiskeridir.no"
+                    "logoPath": "/images/adventure_tech_logo.svg",
+                    "backgroundPath": "/images/client-background-4.svg",
+                    "name": "Adventure Tech",
+                    "text": "Adventure Tech is a Norwegian web development company with adventure travel software.",
```

Comment:
Skulle vi forsøkt noe a.la. dette?
Adventure Tech is a Norwegian tech company providing software for the adventure travel business.

Reply:
Done!